be/src/http/action/stream_load.cpp

```diff
@@ -383,7 +383,7 @@ Status StreamLoadAction::_process_put(HttpRequest* http_req, StreamLoadContext*
     request.formatType = ctx->format;
     request.__set_loadId(ctx->id.to_thrift());
     if (ctx->use_streaming) {
-        auto pipe = std::make_shared<StreamLoadPipe>(1024 * 1024 /* max_buffered_bytes */,
```

Comment:
Should define a constant of `max_buffered_bytes` instead of modifying every place every time